app/Providers/AppServiceProvider.php

```diff
@@ -199,5 +200,24 @@ private function bootCustomValidators()
 
             return false;
         });
+        Validator::extend('device_group_names', function ($attribute, $value): bool {
```

Comment:
This validator can be represented as a regular string instead of adding a custom validator.
`string|unique:device_groups,name`

https://laravel.com/docs/11.x/validation#rule-unique